### ISSUE TITLE
Can't make the image boot

### ISSUE THREAD

**Issue by kingio** [imperador/chromefy#35]
**Device**: TinkerBoard (RK3288 - armv7)
**Device for boot**: 32GB SD Card
**Base image**:  https://chromium.arnoldthebat.co.uk/special//Cx86OS_R60-9592.B-Special.7z
**Recovery image**: ASUS Chromebit CS10  (RK3288)(https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_11021.81.0_veyron-mickey_recovery_stable-channel_mickey-mp-v2.bin.zip)
**TPM**: Caroline

**Steps:**
1. Clone this repo
2. Download the img / bin files
3. Rename **Base image** to `chromiumos_image.img`
4. Run ` sudo bash chromefy.sh chromiumos_image.img chromeos_11021.81.0_veyron-mickey_recovery_stable-channel_mickey-mp-v2.bin caroline.bin`
5. Burn the remaining `chromiumos_image.img` to an SD Card using Etcher.

It when i start the device, it shows nothing, only power led is on :(

**Comment by alesimula** [comment]
Did you make sure the chromium OS images you were using was working on tinkerboard on the first place?

**Comment by kingio** [comment]
Yes, i also tried it with FydeOS build for tinkerboard with the same result.
